packages/cadl-typescript/test/unit/generate/parameters.spec.ts

```diff
@@ -144,4 +144,179 @@ describe("Parameters.ts", () => {
       );
     });
   });
+
+  describe("Array generation", () => {
+    it("string array request generation", async () => {
+      const parameters = await emitParameterFromCadl(`
+      @post op read(): string[];
```

Comment:
```suggestion
      @post op read(body: string[]): void ;
```

Reply:
@kazrael2119 Please notice in `parameters.spec.ts` testing we mainly focus on verifying the inputted parameter so the array data should be in method input position.